tests/support/redis.tcl

```diff
@@ -340,6 +376,16 @@ proc ::redis::redis_read_reply {id fd} {
     }
 }
 
+proc ::redis::redis_read_reply {id fd} {
+    set response [redis_read_reply_logic $id $fd]
```

Comment:
1. log current command in __dispatch__raw__ (connection var, like $::redis::response_interpreters)
2. $::redis::response_interpreters will no longer be a connection var, but rather a table mapping command name to its interpreter
3. here in this function we will find our interpreter by using the command name and said table